static/download_data.html

```diff
@@ -87,6 +87,70 @@ <h2>AddBiomechanics 1.0 Dataset (High Quality Core Data):</h2>
         Project</a>
       <a href="assets/AddBiomechanics_Dataset_Paper.pdf" class="btn btn-secondary btn-lg">Read the Pre-Print</a>
     </p>
+    <style>
+        .code-container {
+            position: relative;
+            margin: 1em 0;
+        }
+
+        .copy-button {
+            position: absolute;
+            top: 10px;
+            right: 10px;
+            padding: 5px 10px;
+            font-size: 0.9em;
+            cursor: pointer;
+            border: none;
+            background-color: #4CAF50;
+            color: white;
+            border-radius: 3px;
+        }
+
+        pre {
+            background-color: #f4f4f4;
+            padding: 1em;
+            border: 1px solid #ccc;
+            border-radius: 5px;
+            overflow-x: auto;
+            font-family: Consolas, Monaco, monospace;
+            white-space: pre-wrap;
+        }
+    </style>
+    
+    <h4>Command-line Download Options</h4>
+    <p>
+      You can download the entire AddBiomechanics Dataset using the following commands. Depending on your download speed, this may
+      take several hours to complete (~4 hours with a 50 Mbps connection).
+    </p>
+    <p>
+      Download via "curl":
+    </p>
+    <p>
+      <div class="code-container">
+        <button class="copy-button" onclick="copyCode(this)">Copy</button>
+        <pre><code>curl -o AddBiomechanicsDataset.zip http://archive.simtk.org/addbiomechanics/addbiomechanics.zip</code></pre>
```

Comment:
Should we use this zip link for the [Download Core Dataset](https://github.com/keenon/AddBiomechanics/blob/2359f44061d444be6e03de7b5507ce279582a179/static/download_data.html#L83) button, or should we keep the Drive link just in case?

Reply:
Maybe it could be a good idea to have both. A button to download from the browser using the link from curl, and another one called "Google Drive" for the drive folders, which allows exploring the dataset and download files separately.

Reply:
The last time we discussed this, we were leaning towards having both. That way users could download everything at once or download only the pieces of the dataset they care about.